compiler/lib/src/main/scala/codegen/CppWriter/ArrayCppWriter.scala

```diff
@@ -477,7 +477,7 @@ case class ArrayCppWriter (
     )
     ) ++
       wrapClassMembersInIfDirective(
-        "\n#if FW_ARRAY_TO_STRING",
+        "\n#if FW_SERIALIZABLE_TO_STRING",
```

Comment:
Can we change the guard to `#ifdef FW_SERIALIZABLE_TO_STRING` per the discussion in the other issue?

Reply:
Yes, can do!

Reply:
I think we decided to implement on/off switches as an ifdef check.

Reply:
This should be resolved in recent commit!

Reply:
Thanks! On further consideration, I reverted this change. Replacing `#if` with `#ifdef` changes the configuration procedure: instead of always defining a symbol to 1 or 0, you have to define or not define a symbol. I'm not sure if this change is what we want. In this PR I think we should be conservative and just address #447. Let's continue to have the discussion about configuration procedure.